src/dataServices/dynamoDbParamBuilder.ts

```diff
@@ -142,9 +144,16 @@ export default class DynamoDbParamBuilder {
     }
 
     static buildPutCreateExportRequest(bulkExportJob: BulkExportJob) {
+        const newItem: any = { ...bulkExportJob };
+        if (newItem.tenantId) {
+            newItem[TENANT_ID_FIELD] = newItem.tenantId;

```

Comment:
There's no need to use `TENANT_ID_FIELD='_tenantId'` here.

_tenantId is prefixed with an underscore because on the resources table we prefix with underscore all "internal"fields to avoid conflicts with FHIR resources fields. 

The bulk export jobs ddb records are entirely our own so we don't need "internal" fields.

simply naming the field `tenantId` makes more sense